Makefile

```diff
@@ -25,9 +25,17 @@ env:: | $(CONDA_ENV_PYTHON)
 	git submodule init
 	git submodule update --init --recursive
 	mkdir -p env/symbiflow
+	wget -q https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/releases/download/v1.1.0/Symbiflow_v1.1.0.gz.run
+	mkdir -p env/quicklogic
+	chmod +x Symbiflow_v1.1.0.gz.run
+	INSTALL_DIR=${PWD}/env/quicklogic ./Symbiflow_v1.1.0.gz.run
+	rm Symbiflow_v1.1.0.gz.run
+	source ./env/quicklogic/conda/bin/activate && pip3 install asciitable terminaltables colorclass "git+https://github.com/antmicro/edalize.git@quicklogic#egg=edalize"
+	sed -i "s/\/home\/kkumar\/symbiflow-arch-defs\/build\/env\/conda\/bin\/python3/\/usr\/bin\/env python3/" ./env/quicklogic/install/bin/python/qlfasm #fix path to interpreter, see https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/issues/16
```

Comment:
I am doubtful on this procedure. What happens is that the conda environment gets generated using a different methodology (instead of using the [make-env repository](https://github.com/SymbiFlow/make-env)).

I understand that quicklogic uses a different yosys and VTR versions, plus other python libraries not present in the original conda requirements.txt. I am unaware whether we can modify the conda configuration file to set two different conda environments, but, if this is doable, we should set the Quicklogic environment using the configuration files.

Reply:
Done, I changed installation to use make-env. Now when you are downloading environment, you need to specify which one: symbiflow or quicklogic:
```
TOOLCHAIN=symbiflow make env
TOOLCHAIN=quicklogic make env
```
By default (when toolchain is not specified explicit before make command) it is only downloading symbiflow.

Reply:
Perfect, looks better now, thanks